daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -40,7 +40,7 @@ spec:
     value: ":"
 ```
 
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
+Make sure to replace `<PATH TO SECRETS FILE>` with the path (relative to the root of the application folder) to the JSON file you just created.
```

Comment:
How does this work if the component directory is not in the application directory, such as the default component folder?  Is the path determined by where `dapr run` is executed or where the component directory lives?

Reply:
Confirmed that it's where `dapr run` is executed.